docs/guides/endpoint_creation.md

```diff
@@ -7,7 +7,8 @@ An example is provided below:
 ```
 model_name = "llama-2-7b.suffix.2023-07-18-12-00-00"
 response = Model.get(model_name)
-while response.status != "READY":
+while response.status.name != "READY":
+    print(response.status.name)
     time.sleep(60)
```

Comment:
wait for 10 seconds? with recent chagnes we should be able to deploy things faster

Reply:
Hm what changes?

Reply:
> what changes

Ray's cold start changes

Reply:
Good to merge this in now?